### ISSUE TITLE
Incorrect syntax /example in Wiki for Event-> datatable.selectrow 

### ISSUE THREAD

**Issue by jaimefuhr** [fiduswriter/Simple-DataTables#188]
The format shown does not work with the latest version.

`datatable.on('datatable.selectrow', function(row, event) {
});`

A single object is passed to the function containing row, and event.  

`datatable.on('datatable.selectrow', function(data) {
var row = data.row;
var event = data.event;
});`

**Comment by johanneswilm** [comment]
@jaimefuhr The format you are showing was only used for a very brief period and should have been changed to the one used in the wiki a week ago in version 4.0.7 to be consistent with all other events. In which distribution format did you find the format you mention?

**Comment by jaimefuhr** [comment]
I am using this CDN:

`https://cdn.jsdelivr.net/npm/simple-datatables@latest`

The comments reveal this to be from 4.0.7.   This CDN was included in a StartBootstrap template.

Looks like it's using an old CDN link?

**Comment by johanneswilm** [comment]
@jaimefuhr Let me know how it goes, especially if the problem is on my side. Version 4.0.7 should have the correct API.

**Comment by jaimefuhr** [comment]
I added a pair to the CDN querystring to force a refresh, but there was no change.

I then copied the source from the CDN into a Javascript file and referenced that.  Same results.   I'm only getting a single object containing the row and event.

When I initialize my table, I have to add 'simpleDatatables.' to DataTable name or I get an unknown reference error

`let ct = document.querySelector("#collections");
                let tblCollections = new simpleDatatables.DataTable(ct);`

whereas the documentation uses new Datatable(tableID).

I'll create a new project with just the simple datatables and see if I get the same results.
